### DISCUSSION TITLE
Oauth error invalid_request: The redirect_uri is not whitelisted

### DISCUSSION THREAD

**Discussion by da-mask** [osiset/laravel-shopify#701] [General]
I've been chasing my tail for a few days on this. I'm not sure why I'm getting this,  `/authenticate` is set as allowed redirection URL. 

Could it be a conflict with Jetstream, which I've implemented on my laravel app?

**Comment by da-mask** [comment]
I was implementing `MustVerifyEmail` in Fortify, so I removed this, but I'm still have an issue with the user without a password.  I'm not sure how I'll handle this in production,

Cheers for the great work, btw

**Reply by gnikyt** [reply]
Is MustVerifyEmail on by default in User models now?

**Reply by da-mask** [reply]
No. I've removed that condition for the moment. I was trying to authenticate my shop app running on my localhost using ngrok, and I thought the issue might be some cross site issue, because the app was using a different url locally to the ngrok provided one. 

But I've pushed the app up to an AWS instance with a https url, and I'm still getting the same issue.  I just get a 400 after being taken to /authenticate . I've done the basic webhooks setup as per the installation instructions, I haven't added anything to the AppUninstalledJob job, could this be the problem?

**Reply by gnikyt** [reply]
ngrok gives issues for many, not sure there. The uninstall job shouldn't cause the 400, it soft deletes the shop and resets it's data.

**Reply by da-mask** [reply]
You know what, I left out the `www` from my url when setting up my Shopify app. Despite showing me up to be an idiot, I thought I better come back here and post the solution, which might save someone else from getting egg on their face.
